Sources/ForageSDK/ForageSDK.swift

```diff
@@ -28,10 +28,6 @@ public class ForageSDK {
             assertionFailure("ForageSDK is not initialized - call ForageSDK.setup() before accessing ForageSDK.shared")
```

Comment:
Change this `guard` to just check to ensure the config exists.

Reply:
5a61927098e171b0e90ad532184623d74f6c23da